### PR TITLE
Import zypper formula

### PR DESCRIPTION
**Pull request by tacerus**
Variant one, _without_ `--committer-date-is-author-date` - I'd prefer this, but it seems GitHub UI does not show author dates, making the changes look more recent than they are.

### CODE REVIEW

File: zypper-formula/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+grains formula
```
Comment (cboltz) [inline zypper-formula/CHANGELOG.rst:1]: s/grains/zypper/

Comment (tacerus) [reply zypper-formula/CHANGELOG.rst:1]: Thanks, I'll go through the files and do some clean ups before finalizing the import.

Comment (cboltz) [reply zypper-formula/CHANGELOG.rst:1]: LICENSE probably also needs an update ;-)

**Comment by cboltz** [comment]
Keeping the original author dates makes more sense.

**Comment by tacerus** [comment]
Thanks for the input, I'll drop the other one then.

File: zypper-formula/CHANGELOG.rst (hunk 1 above)
Comment (tacerus) [reply zypper-formula/CHANGELOG.rst:1]: Hm, do you think the copyright holder should be changed? I figured it'd be best to keep the original one.

**Comment by tacerus** [comment]
I updated the files. Maybe it could do with some modernizing at a later point.

File: zypper-formula/CHANGELOG.rst (hunk 1 above)
Comment (tacerus) [reply zypper-formula/CHANGELOG.rst]: I added a second entry with the recent years, I think that should be good.

**Comment by cboltz** [comment]
Looking at the commits, they all say "authored and commited 3 hours ago" (while I'd expect that some of them are a few years old). It looks like you closed the wrong PR ;-)

**Comment by tacerus** [comment]
Are you sure?

```
$ git branch |grep '^*'
* import/zypper

$ git log --format=fuller HEAD~3 | grep -E '^\w|^$' |head -n21
commit 3be710e721d8690336c0e24784549fc3e66ca240
Merge: 148a6b9 2e822e7
Author:     Daniel Rodríguez <daniel.rodriguez@suse.com>
AuthorDate: Mon Oct 18 15:22:02 2021 +0000
Commit:     Georg Pfuetzenreuter <georg.pfuetzenreuter@suse.com>
CommitDate: Fri May 12 19:56:44 2023 +0200


commit 148a6b9d99ef1739ad4512c8352c83591d8831db
Merge: 8c47ca1 3a4b9e0
Author:     bmwiedemann <bwiedemann@suse.com>
AuthorDate: Thu Feb 4 08:09:00 2021 +0000
Commit:     Georg Pfuetzenreuter <georg.pfuetzenreuter@suse.com>
CommitDate: Fri May 12 19:56:44 2023 +0200


commit 2e822e7ff4a7239375165acf53f0c6a47e1fecf2
Author:     Daniel Rodríguez <daniel.rodriguez@suse.com>
AuthorDate: Mon Oct 18 16:30:31 2021 +0200
Commit:     Georg Pfuetzenreuter <georg.pfuetzenreuter@suse.com>
CommitDate: Fri May 12 19:56:44 2023 +0200
```

**Comment by tacerus** [comment]
I think that's what I meant in my PR message above - the GitHub UI does not display author dates at all, giving a wrong impression to when the change was made.

**Comment by cboltz** [comment]
Indeed, if I checkout your branch, the original dates are shown in git log. With --format=fuller I see that the CommitDate is today, which is fine.

So this is only an issue on github. Confusing when reviewing the PR, but besides that not a real issue. Therefore feel free to keep old AuthorDate and today as CommitDate, and merge this PR as is.

(For completeness, if someone reads this PR later: https://github.com/openSUSE/salt-formulas/pull/7/commits displays the original dates because you used the original AuthorDate as CommitDate. This might look better on github, but is not really correct.)

**Pull request merged**